server/src/main/java/io/envoyproxy/controlplane/server/limits/ManualFlowControl.java

```diff
@@ -0,0 +1,78 @@
+package io.envoyproxy.controlplane.server.limits;
+
+import io.envoyproxy.envoy.api.v2.DiscoveryResponse;
+import io.grpc.stub.ServerCallStreamObserver;
+import java.util.concurrent.atomic.AtomicBoolean;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ManualFlowControl implements FlowControl<DiscoveryResponse> {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(ManualFlowControl.class);
+
```

Comment:
nit: other code in this codebase avoids newlines between fields

Reply:
Sort of yes, but not consistently :-) Check SimpleCache for example. Of course I'll remove newlines.